.github/workflows/integration.yml

```diff
@@ -9,188 +9,27 @@ on:
-    steps:
-      - uses: actions/checkout@v2
-      - name: Prep-Server-DB
-        run: make server_tests
-  
-  Prep-Kotlin-Core:
-    runs-on: [self-hosted, dockerized]
-    needs: [Core, Server]
-    env:
-      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-    steps:
-      - uses: actions/checkout@v2
-      - name: Prep-Kotlin-Core
-        run: make kotlin_interface_tests
-  
-  Prep-Swift-Core:
-    runs-on: [self-hosted, dockerized]
-    needs: [Core, Server]
-    env:
-      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-    steps:
-      - uses: actions/checkout@v2
-      - name: Prep-Swift-Core
-        run: make swift_interface_tests
-
-  Run-Dev-Stack:
-    runs-on: [self-hosted, dockerized]
-    needs: [Prep-Core-Server-Db, Prep-Server-DB, Prep-Kotlin-Core, Prep-Swift-Core]
-    env:
-      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-    steps:
-      - uses: actions/checkout@v2
-      - name: Spin up dev stack
-        run: make dev_stack_run
-
-  Core-Server-Db:
-    runs-on: [self-hosted, dockerized]
-    needs: Run-Dev-Stack
-    env:
-      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-    steps:
-      - uses: actions/checkout@v2
-      - name: Core-Server-DB Tests
-        run: make core_server_tests_run
-
-  Kotlin-Core:
-    runs-on: [self-hosted, dockerized]
-    needs: Run-Dev-Stack
-    env:
-      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-    steps:
-      - uses: actions/checkout@v2
-      - name: Kotlin-Core
-        run: make kotlin_interface_tests_run
-
-  Swift-Core:
-    runs-on: [self-hosted, dockerized]
-    needs: Run-Dev-Stack
-    env:
-      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-    steps:
-      - uses: actions/checkout@v2
-      - name: Swift-Core
-        run: make swift_interface_tests_run
-
-  Server-Logs:
-    runs-on: [self-hosted, dockerized]
-    needs: [Core-Server-Db, Kotlin-Core, Swift-Core]
-    if: always()
-    env:
-      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-    steps:
-      - uses: actions/checkout@v2
-      - name: Get Server Logs
-        run: make get_server_logs
-        if: always()
-
-  Clean-Up:
-    runs-on: [self-hosted, dockerized]
-    needs: [Server-Logs]
-    if: always()
-    env:
-      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-    steps:
-      - uses: actions/checkout@v2
-      - name: Tear Down Stack
-        run: make kill_dev_stack
-        if: always()
+    - name: Check Workspace Fmt
+      run: cargo run -p lockbook-dev -- check-workspace-fmt
+    - name: Check Workspace Clippy
+      run: cargo run -p lockbook-dev -- check-workspace-clippy
+    - name: Run Server
+      run: cargo run -p lockbook-dev -- run-server
+    - name: Run Rust Tests
+      run: cargo run -p lockbook-dev -- run-rust-tests
+    - name: Run Android Fmt
+      run: cargo run -p lockbook-dev -- check-android-fmt
+    - name: Run Android Lint
+      run: cargo run -p lockbook-dev -- check-android-lint
+    - name: Run Swift Tests
+      run: cargo run -p lockbook-dev -- run-swift-tests
```

Comment:
for now comment this out, I will fix these shortly, but that would make this a green build, and we can require it for pr's immediately

Reply:
never mind actually, we should just fix these asap